lib/sycamore/sycamore/tests/integration/query/execution/test_sycamore_query.py

```diff
@@ -80,7 +80,7 @@ def test_vector_search(self, query_integration_test_index: str, codegen_mode: bo
             "give me some wind related incidents",
             query_integration_test_index,
             schema,
-            natural_language_response=False,
+            natural_language_response=True,
```

Comment:
Not sure about this one. I think this is a bug as I think it should actually be `False` for this test.